earthaccess/search.py

```diff
@@ -66,11 +96,32 @@ def hits(self) -> int:
 
         try:
             response.raise_for_status()
-        except exceptions.HTTPError as ex:
+        except requests.exceptions.HTTPError as ex:
             raise RuntimeError(ex.response.text)
 
         return int(response.headers["CMR-Hits"])
 
+    def get(self, limit: int = 2000) -> List[Any]:
```

Comment:
Given that we're dealing only with umm-json, we can use the specific `DataCollection` type instead of `Any`:

```suggestion
    def get(self, limit: int = 2000) -> List[DataCollection]:
```